README.md

```diff
@@ -1,3 +1,5 @@
 # Some stuff
 
 Hey, it's me.
+
```

Comment:
Hey, we don't need this new line. Please remove

Reply:
Fixed it, took 2000 hours